webpack.config.js

```diff
@@ -45,9 +49,9 @@ module.exports = (env, options) => {
     output: {
       //  NOTE: this is a little different from usual Springload setup due to different directory structure of this site.
       path: path.resolve(__dirname, outputPath),
-      publicPath: '/static/dist/', // For some reason this adds an extra `/dist/` layer in the output, but only for font files. TODO resolve.
+      publicPath: '/static/dist/',
       filename: '[name].js', // No filename hashing, Django takes care of this
-      chunkFilename: '[id]-[chunkhash].js',
+      chunkFilename: '[name]-[chunkhash].js',
       clean: true,
```

Comment:
This clean: true is all we'd normally need, I think. Something about this project seems to make that not work properly for all chunks 🤔